data/src/main/java/io/gnosis/data/backend/GatewayApi.kt

```diff
@@ -14,43 +15,71 @@ import java.util.*
 
 interface GatewayApi {
 
-    @GET("/v1/balances/supported-fiat-codes")
-    suspend fun loadSupportedCurrencies(): List<String>
+    @GET("/{chainId}/v1/balances/supported-fiat-codes")
+    suspend fun loadSupportedCurrencies(
+        @Path("chainId") chainId: Int = 4
```

Comment:
default value could come from build constants (BuildConfig.CHAIN_ID). I think we even have it already